### ISSUE TITLE
Hide Private Subreddits?

### ISSUE THREAD

**Issue by benjmd** [Tanza3D/reddark#78]
We have 'hide public', can we 'hide private' as an option to see who the hold-outs are?
Might encourage users to post in the online subreddits and ask if they'd be willing to participate.

**Comment by Mewtwo2387** [comment]
The ones listed on the website are already those that announced they will participate. Those that are public either haven't done it yet or the website didn't register

**Comment by benjmd** [comment]
Yep am aware, right now it takes a fair bit of scrolling to find them among the private subs vs a one-click button option to show/hide.
